src/App.tsx

```diff
@@ -1,29 +1,50 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useState , FC} from 'react';
 import './App.scss';
-import { getAllCurrencies } from 'src/service/getAllCurrencies';
-import { ListCurrencies } from 'src/components/List-currencies/List-currencies';
+import { getAllCurrencies, getConvertedValue } from 'src/service/CurrencyService';
+import { ListCurrencies } from 'src/components/ListCurrencies/ListCurrencies';
 
 
 function App() {
   
-  const [allCurrencies, setAllCurrencies] = useState([''])
-  const [leftInputValue, setLeftInputValue] = useState('')
-  const [rightInputValue, setRightInputValue] = useState('')
-  const [isReversed, setIsReversed] = useState(false)
+  const [allCurrencies, setAllCurrencies] = useState<string[]>([''])
+  const [leftInputValue, setLeftInputValue] = useState<string|number>('')
+  const [rightInputValue, setRightInputValue] = useState<string|number>('')
+  const [isReversed, setIsReversed] = useState<boolean>(false)
+  const [inputCurrency, setInputCurrency] = useState('')
+  const [outputCurrency, setOutputCurrency] = useState('')
```

Comment:
I can't match inputCurrency/output with left and right input value. Use the same logic for naming